src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -255,17 +324,19 @@ static bool process_resolver_result_locked(
     void* arg, const grpc_channel_args& args, const char** lb_policy_name,
     grpc_core::RefCountedPtr<LoadBalancingPolicy::Config>* lb_policy_config) {
   channel_data* chand = static_cast<channel_data*>(arg);
-  chand->have_service_config = true;
+  chand->received_resolver_result.Store(true, grpc_core::MemoryOrder::RELEASE);
```

Comment:
I have two rather stupid questions:
- Is `received_resolver_result` initialized anywhere?
- Which fields can we access when it's set?